### ISSUE TITLE
Filterbar design

### ISSUE THREAD

**Issue by Phyks** [Phyks/ampache_react#8]
How should the filterbar work?

For now, it filters the displayed items.

I'd like it to be more like a global search bar and perform a search on enter, with some syntax _à la_ Github issue bar.

**Comment by Gano16** [comment]
I'm not sure what the filterbar is doing on the current version, because it seems me it's working a little strangely...

Nevertheless, if it is not already the case, if displayed items are on several pages, I think it should filter on all the pages (doesn't seem to be the case on the tested version).

Otherwise, I think an _à la_ Github issue bar would be a great idea.

I think maybe both proposals can work together, the first one without pressing enter, and the second one when pressing it.

**Comment by Phyks** [comment]
Indeed. An idea I had is to do the following whenever some text is entered in the filterbar:
1. Filter the already loaded elements, and display them.
2. Query the Ampache server for extra results (in the remaining pages).
3. Whenever we receive them, add them to the current view.

This requires a bit of work though.

**Comment by Phyks** [comment]
Also "Filter..." placeholder should be replaced by a random item from the catalog.
